CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please add the names of the contributors

Reply:
Resolved